### PR TITLE
hardening/294_rename_packages

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue https://github.com/telefonicaid/fiware-cygnus/issues/294
* 100% unit tests passed:

```
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.037 sec
Results :
Tests run: 45, Failures: 0, Errors: 0, Skipped: 0
```

* (unofficial) e2e tests not required.
* Assignee @fgalan but LGTM from @gtorodelvalle is also required.

### CODE REVIEW

File: src/main/java/com/telefonica/iot/cygnus/channels/CygnusFileChannel.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
  *
  * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
```
Comment (fgalan) [inline src/main/java/com/telefonica/iot/cygnus/channels/CygnusFileChannel.java]: I'd suggest to use this PR to change also license header, eg. use fiware-cygnus as project name.

Comment (frbattid) [inline src/main/java/com/telefonica/iot/cygnus/channels/CygnusFileChannel.java]: Good catch

**Comment by gtorodelvalle** [comment]
Hi guys! After checking out the ```hardening/294_rename_packages``` branch, I found that there are still some references to ```es.tid.fiware```(see below) I guess we would also want to update or get rid of. Don't we, @frbattid ? :)
fiware-references

**Comment by frbattid** [comment]
Good catch, I'll use the replacement utility from my IDE in order to make a global replacement, the same way I've globally replaced all the occurences of "fiware-connectors" (401 occurences wooow... be prepared for the next commit :)).

**Comment by gtorodelvalle** [comment]
Hell yeah!!! The mother of all the commits... \o/

**Comment by frbattid** [comment]
I think now all references to es.tid.fiware.* have been replaced... Very ugly PR, indeed :(

**Comment by fgalan** [comment]
LGTM (passing the ball to @gtorodelvalle for final LGTM + merge + delete branch).

**Comment by gtorodelvalle** [comment]
In IDEs we believe (I checked there are no ```es.tid.fiware```references in the project)... :package: 

LGTM!!! ;) Thanks @frbattid !

**Pull request merged**